aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -720,6 +712,55 @@ module aptos_token::token {
         token_data.uri = uri;
     }
 
+    public fun mutate_tokendata_royalty(creator: &signer, token_data_id: TokenDataId, royalty: Royalty) acquires Collections {
```

Comment:
we need a public function to create Royalty. Otherwise, other contracts won't be able to generate this parameter

Reply:
good point - just added!